README.md

```diff
@@ -149,70 +118,47 @@ Some examples:
 
 # capture 4mb of video from camera
 ./ankerctl.py pppp capture-video -m 4mb output.h264
+
+# select printer to use when you have multiple
+./ankerctl.py -p <index> #index starts at 0 and goes up to the number of printers you have
 ```
 
-## Webserver
+### Web Interface
 
-ankerctl can also be used as a webserver to allow slicers like prusaslicer to print directly to the printer. 
+1. Start the webserver by running one of the following commands in the folder you placed ankerctl in. You’ll need to have this running whenever you want to use the web interface or send jobs to the printer via a slicer:
 
-![Screenshot of prusa slicer](/static/img/setup/prusaslicer-2.png?raw=true "Screenshot of prusa slicer")
+   Docker Installation Method:
```

Comment:
Let's not mention `docker-compose` here, given high likely it is to give us trouble right now.

Advanced users should be able to figure it out for themselves for now.